processing/data_collection/gazette/spiders/df_brasilia.py

```diff
@@ -0,0 +1,80 @@
+import re
+import json
+import itertools
+import dateparser
+import requests
+
+from datetime import datetime
+from scrapy import Request, Spider
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class DfBrasiliaSpider(BaseGazetteSpider):
+    MUNICIPALITY_ID = '5300108'
+    GAZETTE_URL = 'http://dodf.df.gov.br/listar'
+
+    MONTHS = [
+        '12_Dezembro',
+        '11_Novembro',
+        '10_Outubro',
+        '09_Setembro',
+        '08_Agosto',
+        '07_Julho',
+        '06_Junho',
+        '05_Maio',
+        '04_Abril',
+        '03_Março',
+        '02_Fevereiro',
+        '01_Janeiro'
+    ]
+    YEARS = ['2018', '2017', '2016', '2015']
+    MONTHS_YEARS = itertools.product(MONTHS, YEARS)
+
+    DATE_REGEX = r'.+\/DODF [0-9]+ ([0-9]{2}-[0-9]{2}-[0-9]{4})(.*)$'
+    EXTRA_EDITION_TEXT = "EDICAO EXTR"
+    PDF_URL = 'http://dodf.df.gov.br/index/visualizar-arquivo/?pasta={}&arquivo={}'
+
+    allowed_domains = ['dodf.df.gov.br']
+    name = 'df_brasilia'
+
+    def start_requests(self):
+        for month, year in self.MONTHS_YEARS:
+            response = requests.get(f'{self.GAZETTE_URL}?dir={year}/{month}')
```

Comment:
Try to use `scrapy.Request` instead of a blocking third-party library. I think it will be better to create a new `parse_month` method to handle the response of these requests, and there include the new requests for each day.

Reply:
Done @rennerocha! 👍